### PR TITLE
chore: Move docs from old repo

### PR DESCRIPTION
**Pull request by FrankEssenberger**
## Context

I finally moved the docs from the Old repo to the new OS one. I hope I did not miss any internal link. 
The overview document on the tutorials I would remove. What do you think?

### CODE REVIEW

File: knowledge-base/architectural-decisions-records/circuit-breaker-options.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Circuit Breaker Options
```
Comment (marikaner) [inline knowledge-base/architectural-decisions-records/circuit-breaker-options.md:1]: The directory for this should be `architectural-decision-records` (or maybe even rename it to `adr`)

File: knowledge-base/tutorials.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+I think this is outdated and with the github.io we do not need it
```
Comment (marikaner) [inline knowledge-base/tutorials.md:1]: I think it is still valid. We will keep those tutorials up, the documentation is a technical addition to the tutorials, which are more a step by step guide to achieve a goal.

File: knowledge-base/implementation-documentation/connectivity-service-on-prem-destinations.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+     "onpremise_proxy_host": "connectivityproxy.internal.cf.sap.hana.ondemand.com",
+     "onpremise_proxy_port": "20003",
+     "onpremise_proxy_rfc_port": "20001",
+     "onpremise_socks5_proxy_port": "20004",
+     "tenantid": "a89ea924-d9c2-4eab-84fb-3ffcaadf5d24",
+     "tenantmode": "dedicated",
+     "uaadomain": "authentication.sap.hana.ondemand.com",
+     "url": "https://s4sdk.authentication.sap.hana.ondemand.com",
+     "verificationkey": "-----BEGIN PUBLIC KEY-----MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAx/jN5v1mp/TVn9nTQoYVIUfCsUDHa3Upr5tDZC7mzlTrN2PnwruzyS7w1Jd+StqwW4/vn87ua2YlZzU8Ob0jR4lbOPCKaHIi0kyNtJXQvQ7LZPG8epQLbx0IIP/WLVVVtB8bL5OWuHma3pUnibbmATtbHIi0kyNtJXQvQ7LZPG8epQLbx0IIP/WLVVVtB8bL5OWuHma3pUnibbmATtbOh5LksQ2zLMngEjUF52JQyzTpjoQkahp0BNe/drlAqO253keiY63FL6belKjJGmSqdnotSXxB2ym+HQ0ShaNvTFLEvi2+ObkyjGWgFpQaoCcGq0KX0y0mPzOvdFsNT+rBDQIDAQAB-----END PUBLIC KEY-----",
+     "xsappname": "clone714465bab477466ba097e647388e61b4!b2499|connectivity!b137"
```
Comment (marikaner) [inline knowledge-base/implementation-documentation/connectivity-service-on-prem-destinations.md:36]: O.o

**Review by marikaner — CHANGES_REQUESTED** [review]
I think you missed something in the vcaps there

File: knowledge-base/implementation-documentation/connectivity-service-on-prem-destinations.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+    ],
+    "volume_mounts": []
+   }
+  ],
+ },
+ ...
+}
+```
+
+`onpmremise_proxy_host` and `onpremise_proxy_port` give you the URI that a request has to be tunneled through, in this case `http://connectivityproxy.internal.cf.sap.hana.ondemand.com:20003`.
```
Comment (marikaner) [inline knowledge-base/implementation-documentation/connectivity-service-on-prem-destinations.md:57]:
```suggestion
`onpremise_proxy_host` and `onpremise_proxy_port` give you the URI that a request has to be tunneled through, in this case `http://connectivityproxy.internal.cf.sap.hana.ondemand.com:20003`.
```

File: knowledge-base/architectural-decisions-records/circuit-breaker-options.md (hunk 1 above)
Comment (FrankEssenberger) [reply knowledge-base/architectural-decisions-records/circuit-breaker-options.md:1]: renamed

File: knowledge-base/tutorials.md (hunk 2 above)
Comment (FrankEssenberger) [reply knowledge-base/tutorials.md:1]: Yes, I meant that we reference the tutorials on the github.io https://sap.github.io/cloud-sdk/docs/overview/getting-started
So I wonder if it makes sense to keep a markdown in the reop?

Comment (marikaner) [reply knowledge-base/tutorials.md:1]: Ah no, you can delete it, good point.

**Review by marikaner — APPROVED** [review]
I didn't read all of it right now, but looks good. Please still remove the tutorial reference

**Comment by marikaner** [comment]
@FrankEssenberger BTW (minor): I noticed that you are often not using `chore:` in the PR title but `[Chore]`. I now changed it to `chore:`, please use the convention if possible.

**Comment by FrankEssenberger** [comment]
> @FrankEssenberger BTW (minor): I noticed that you are often not using `chore:` in the PR title but `[Chore]`. I now changed it to `chore:`, please use the convention if possible.

Ah I thougth that the brackets are the convention... ok I will try to remember. Same for fix: and feature: I guess.

**Comment by marikaner** [comment]
> > @FrankEssenberger BTW (minor): I noticed that you are often not using `chore:` in the PR title but `[Chore]`. I now changed it to `chore:`, please use the convention if possible.
> 
> Ah I thougth that the brackets are the convention... ok I will try to remember. Same for fix: and feature: I guess.

Yes! 😄

**Pull request merged**